core/capabilities/remote/target/request/client_request_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
It looks like std lib errors will suffice?
```suggestion
	"errors"
```